src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -209,6 +253,31 @@ limitations under the License.
             help="Remove MTU">
             <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} 9100 </ACTION>
         </COMMAND>
+
+    <COMMAND
+        name="channel-group"
+        help="Configure port channel parameters"
+        ptype="SUBCOMMAND"
+        mode="subcommand"/>
+    <COMMAND
```

Comment:
Same here, make command as "channel-group" and "mode", "active" as nested PARAMs.

Reply:
fixed